scrapers/nv/bills.py

```diff
@@ -228,11 +229,15 @@ def add_actions(self, bill, chamber):
                 elif "Governor" in action:
                     actor = "executive"
 
-                action_type = None
+                action_type = []
                 for pattern, atype in ACTION_CLASSIFIERS:
                     if re.search(pattern, action, re.IGNORECASE):
-                        action_type = atype
-                        break
+                        if isinstance(atype, str):
```

Comment:
We could go further and turn that into a list comprehension, but I don't think there's enough benefit to outweigh the decrease in readability.

Reply:
you know looking at this more i'm just going to edit the classifier so that it always returns a list, it's more clear than the mixed type stuff. 

Otherwise your edits make sense, will push a new version later. Thanks!